include/dt-bindings/pinctrl/esp32-pinctrl.h

```diff
@@ -10,6 +10,14 @@
 #ifndef INC_DT_BINDS_PINCTRL_ESP32_PINCTRL_HAL_H_
 #define INC_DT_BINDS_PINCTRL_ESP32_PINCTRL_HAL_H_
 
+/* DAC_CH1 */
+#define DAC_CH1_GPIO25 \
+	ESP32_PINMUX(25, ESP_NOSIG, ESP_DAC1_OUT)
```

Comment:
This was automatically generated. I don't think it will be an issue to keep this in here.